maat-court-data-api/src/main/java/gov/uk/courtdata/assessment/service/PassportAssessmentService.java

```diff
@@ -0,0 +1,61 @@
+package gov.uk.courtdata.assessment.service;
+
+import gov.uk.courtdata.assessment.impl.PassportAssessmentImpl;
+import gov.uk.courtdata.assessment.mapper.PassportAssessmentMapper;
+import gov.uk.courtdata.dto.PassportAssessmentDTO;
+import gov.uk.courtdata.entity.PassportAssessmentEntity;
+import gov.uk.courtdata.exception.MAATCourtDataException;
+import gov.uk.courtdata.model.assessment.CreatePassportAssessment;
+import gov.uk.courtdata.model.assessment.UpdatePassportAssessment;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class PassportAssessmentService {
+
+    private final PassportAssessmentImpl passportAssessmentImpl;
+    private final PassportAssessmentMapper passportAssessmentMapper;
+
+    public PassportAssessmentDTO find(Integer passportAssessmentId) {
+        PassportAssessmentEntity passportAssessmentEntity = passportAssessmentImpl.find(passportAssessmentId);
+        return buildPassportAssessmentDTO(passportAssessmentEntity);
+    }
+
+    @Transactional(rollbackFor = MAATCourtDataException.class)
+    public PassportAssessmentDTO update(UpdatePassportAssessment updatePassportAssessment) {
+        log.info("Update Passport Assessment - Transaction Processing - Start");
+        PassportAssessmentDTO passportAssessmentDTO =
+                passportAssessmentMapper.updatePassportAssessmentToPassportAssessmentDTO(updatePassportAssessment);
+        log.info("Updating existing passport assessment record");
+        PassportAssessmentEntity passportAssessmentEntity = passportAssessmentImpl.update(passportAssessmentDTO);
```

Comment:
There needs to be a check here to determine whether the passport assessments status is marked as complete, as then it should not be modified.